packages/blocks/src/api/raw-handling/test/paste-handler.js

```diff
@@ -61,6 +61,26 @@ const tableWithHeaderFooterAndBodyUsingRowspan = `
 	</tfoot>
 </table>`;
 
+const markdownTableHTML = `
```

Comment:
```suggestion
const tableWithCellAlignments = `
```

This unit test is not limited to the markdown file.